package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "codemirror-lang-n8n-sql",
-  "version": "1.0.0",
+  "name": "@n8n/codemirror-lang-n8n-sql",
+  "version": "1.0.1",
```

Comment:
Do we need to update the version if this package has never been published before?

Reply:
Not sure. Makes sense using `1.0.0`